lms/lmsweb/views.py

```diff
@@ -357,6 +357,31 @@ def exercises_page():
     )
 
 
+@webapp.route('/exercises/<tagname>')
```

Comment:
Maybe `/exercises/tag/<tagname>`? Or `/tags/exercise/_` in case we'll want to tag other things in the future (users, maybe?)